sdk/monitor/azingest/build.go

```diff
@@ -3,6 +3,7 @@
 
 //go:generate autorest ./autorest.md
 //go:generate rm ./models_serde.go
+//go:generate rm ./models.go
 //go:generate rm ./constants.go
```

Comment:
```suggestion
```
Need to remove this line. Tried generating the code locally and received this error:
`rm: cannot remove './constants.go': No such file or directory`

Reply:
Removed.